cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedStreamSizeScheduleStore.java

```diff
@@ -315,7 +315,6 @@ public void apply() throws Exception {
                                                       AbstractSchedulerService.scheduleIdFor(programId, programType,
                                                                                              scheduleName)));
           table.put(rowKey, columns, values);
-          LOG.debug("Updated schedule {} with columns {}, values {}", scheduleName, columns, values);
```

Comment:
I removed it on purpose as this debug line looks not helpful. It prints bytes and pollutes the logs